app/src/main/cpp/skyline/services/nvdrv/types.h

```diff
@@ -0,0 +1,96 @@
+// SPDX-License-Identifier: MIT OR MPL-2.0
+// Copyright © 2021 Skyline Team and Contributors (https://github.com/skyline-emu/)
+
+#pragma once
+
+#include <common.h>
+
+namespace skyline::service::nvdrv {
+    using FileDescriptor = i32;
+    constexpr FileDescriptor InvalidFileDescriptor{-1};
+
+    /**
+     * @brief Holds the permissions for an nvdrv instance
+     */
```

Comment:
I think this comment is sort of pointless, nothing you can't guess from the title. Same for `SessionContext` below this.